sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -875,6 +912,30 @@ private void setDefaultTransmissionTarget(String transmissionTargetToken) {
         }
     }
 
+    /**
+     * Set trigger interval. The interval should be between 3 seconds and 1 day.
+     * Should be called before the service is started.
+     *
+     * @param seconds Set latency of sending events to Analytics.
+     * @return succeed <code>true</code> if set succeed, <code>false</code> otherwise.
+     */
+    private boolean setInstanceTransmissionInterval(int seconds) {
+        if (mChannel != null) {
+            AppCenterLog.error(LOG_TAG, "Transmission interval should be set before the service is started.");
+            return false;
+        }
+        if (seconds < MINIMUM_TRANSMISSION_INTERVAL_IN_SECONDS || seconds > MAXIMUM_TRANSMISSION_INTERVAL_IN_SECONDS) {
+            AppCenterLog.error(LOG_TAG, String.format(Locale.getDefault(),
+                    "The transmission interval is invalid. The value should be between %d seconds and %d seconds (%d day)",
```

Comment:
The doc says the range of valid value. I don't think we need to print a log with details.

Reply:
I'd prefer to say "is not valid" instead of "is invalid" but not a big deal.